### DISCUSSION TITLE
Packages 403 error with full access PAT

### DISCUSSION THREAD

**Discussion by DannyRichardsonWG** [community/community#53226] [Packages]
### Select Topic Area

Bug

### Body

We’re currently having a problem where we can’t access our nuget repo using our Personal Access Tokens and are getting a 403 error when trying to access. 

The system previously was working correctly and as far as I know nothing has changed on our side. It is affecting multiple devs. 

A PAT has been created, for testing it has been assigned every permission available. 

![image](https://user-images.githubusercontent.com/86000734/233095109-b1876c59-5222-464d-a2c6-51e24f238d6e.png)

The Package Source is declared in NuGet.config called “github”

Opening a browser and navigating to https://nuget.pkg.github.com/Witherslack-Group-Developers/index.json 
Enter username and PAT
It returns the JSON 

So the PAT seems to work. 

However when trying to list the sources via command line it gives a forbidden exception

nuget search -Source github

![image](https://user-images.githubusercontent.com/86000734/233095781-71414390-d631-42ed-af7d-36c229fd92b9.png)

So it looks like the token authenticates but is failing to the authorised to look at the packages even though all permissions have been assigned to the PAT. 

Any ideas on what might have changed?

**Comment by wittml** [comment]
Do you need to click the Configure SSO button? If you're using SSO?

**Reply by ulayera-mbww** [reply]
This worked for me, I usually do it for new tokens, but didn't when refreshing an expired one. Thanks!

**Comment by DannyRichardsonWG** [comment]
Thanks for the response @wittml 

This turned out to be an issue with whitelisting the company IP. Turning the whitelist off stopped the error happening. I re-enabled it a couple of days later and things carried on working so I'm not sure what was going wrong. I spoke with support who said make sure any whitelist is set up at the enterprise and organisation level though at the time of it becoming an issue those settings hadn't been updated in a long time.

Sometimes "Turn it off and on again" really is the solution ^^
